variables.tf

```diff
@@ -42,6 +42,18 @@ variable "aws_account_password_policy" {
   description = "AWS account password policy parameters for the audit, logging and master account"
 }
 
+variable "aws_auditmanager_config" {
```

Comment:
```suggestion
variable "aws_auditmanager" {
```

Reply:
What do you think about renaming this to aws_auditmanager? I was scrolling through the other vars and we never put "config" behind it, like aws_guardduty and aws_security_hub

Reply:
Agree, will update accordingly